src/ucp/core/ucp_am.c

```diff
@@ -26,6 +26,18 @@
 #define UCP_AM_SHORT_REPLY_MAX_SIZE  (UCS_ALLOCA_MAX_SIZE - \
                                       sizeof(ucs_ptr_map_key_t))
 
+#define UCP_WORKER_GET_EP_BY_ID(_worker, _ep_id, _str, _action) \
+    ({ \
+         ucp_ep_h __ep = ucp_worker_get_ep_by_id(_worker, _ep_id); \
```

Comment:
why __ep and not _ep?

Reply:
and i think ucp_worker_get_ep_by_id fails on assert if NULL?

Reply:
no, it is ` ucs_assertv((ep == NULL) || (ep->worker == worker)`